appium/tests/specs/mock/setup/RespectsDisallowAttesterSearchForDomainsOnPreDomainBasic.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { MockApi } from 'api-mocks/mock';
+import {
+    SplashScreen,
+} from '../../../screenobjects/all-screens';
+import {attesterPublicKeySamples} from "../../../../api-mocks/apis/attester/attester-endpoints";
+import SetupKeyScreen from "../../../screenobjects/setup-key.screen";
+import MailFolderScreen from "../../../screenobjects/mail-folder.screen";
+import NewMessageScreen from "../../../screenobjects/new-message.screen";
+
+describe('SETUP: ', () => {
+
+  it('respects disallow_attester_search_for_domains on a per-domain basis', async () => {
+
+    const mockApi = new MockApi();
+    mockApi.fesConfig = { clientConfiguration: {flags: ["NO_PRV_CREATE", "NO_PRV_BACKUP", "NO_ATTESTER_SUBMIT", "PRV_AUTOIMPORT_OR_AUTOGEN", "FORBID_STORING_PASS_PHRASE"], key_manager_url: "https://ekm.flowcrypt.com", disallow_attester_search_for_domains: ["disabled.test"]}};
+    mockApi.attesterConfig = {servedPubkeys: {'attester@disabled.test': attesterPublicKeySamples.valid, 'attester@enabled.test': attesterPublicKeySamples.valid}};
```

Comment:
Same here:
```js
mockApi.attesterConfig = {
    servedPubkeys: {
        'attester@disabled.test': attesterPublicKeySamples.valid, 
        'attester@enabled.test': attesterPublicKeySamples.valid
    }
};
```

Reply:
Is it right that both emails have `attesterPublicKeySamples.valid` key?

Reply:
yes - this test is supposed to start with two public keys available for two different recipients on the key server, and only one of them will be actually found, because searching of one of them is disabled with client configuration of the app.

It's ok that it's actually the same key used in the tests - the test tests whether it can be found by searching email on domain A vs on domain B.